pkg/i2gw/providers/common/utils.go

```diff
@@ -99,6 +101,10 @@ func NameFromHost(host string) string {
 	return step2
 }
 
+func HTTPRouteName(ingressName, host string) string {
+	return fmt.Sprintf("%s-%s", ingressName, NameFromHost(host))
+}
```

Comment:
Should we name this function in a more generic term? I think we will probably use it for other route types in the future

Reply:
+1 
```suggestion
func RouteName(ingressName, host string) string {
	return fmt.Sprintf("%s-%s", ingressName, NameFromHost(host))
}
```

Reply:
Yeah, it makes sense. I changed it to `RouteName` @LiorLieberman